x/epochs/genesis_test.go

```diff
@@ -96,7 +96,7 @@ func TestEpochsInitGenesis(t *testing.T) {
 	require.Equal(t, epochInfo.StartTime.UTC().String(), now.UTC().String())
 	require.Equal(t, epochInfo.Duration, time.Hour*24)
 	require.Equal(t, epochInfo.CurrentEpoch, int64(0))
-	require.Equal(t, epochInfo.CurrentEpochStartTime.UTC().String(), ctx.BlockTime().UTC().String())
+	require.Equal(t, epochInfo.CurrentEpochStartTime.UTC().String(), time.Time{}.String())
```

Comment:
At genesis, epoch start time should be at an empty state as the initial value is given in BeginBlocker